File: twoSum.js
/**
 * @param {number[]} numbers
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (numbers, target) {
  let left = 0;
  let right = numbers.length - 1;

  while (numbers[left] + numbers[right] !== target) {
      let curr = numbers[left] + numbers[right]
      if (curr < target)
          left++;
      if (curr > target)
          right--;
  }

  return [left + 1, right + 1]
};

File: twoSum.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        firstIdx = 0
        lastIdx = len(numbers)-1
        curr = numbers[firstIdx] + numbers[lastIdx]
        while curr != target:
            if curr < target:
                firstIdx+=1
            elif curr > target:
                lastIdx-=1
            curr = numbers[firstIdx] + numbers[lastIdx]
        return [firstIdx + 1, lastIdx + 1]
        